justfile

```diff
@@ -191,6 +191,11 @@ create platform=default_platform:
         "AKS-PEER-SNP")
             just upload-image
 
+            # Populate Terraform variables.
+            echo "name_prefix = \"$azure_resource_group\"" > infra/azure-peerpods/just.auto.tfvars
+            echo "image_resource_group_name = \"$azure_resource_group\"" >> infra/azure-peerpods/just.auto.tfvars
```

Comment:
Not sure tbh, will uplosi create the rg the image should be uploaded to? Otherwise we need to create that for the dev at some point.

Reply:
No, right, it won't, I forgot about that. I'll fix that.